operator/v1/types_descheduler.go

```diff
@@ -0,0 +1,52 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +genclient:nonNamespaced
```

Comment:
@soltysh I also wanted to ask about this, currently the descheduler operator is namespaced but should we consider switching it to nonNamespaced in prep for GA? The operator code currently assumes/depends on it being deployed specifically only to the `openshift-kube-descheduler-operator` namespace

Reply:
I think we want non-namespaced, I doubt we'll be running more than a single instance of the descheduler. I'm also fine with the current requirement to run in a particular namespace. I don't see any reason why we would not.